pylint_nautobot/use_fields_all.py

```diff
@@ -0,0 +1,46 @@
+"""Check for CharField's or TextField's on models where null=True and blank=True."""
+from astroid import Assign
+from astroid import AssignName
+from astroid import ClassDef
+from astroid import Const
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+_META_CLASSES = (
+    "nautobot.utilities.tables.BaseTable.Meta",
+    # TBD: Add support for other Meta classes
+    "nautobot.extras.forms.NautobotModelForm.Meta",
+)
+
+
+class NautobotUseFieldsAll(BaseChecker):
```

Comment:
Just making sure that we are purposeful, do we want to:

* Create one check for all types (serializer, forms, tables, etc)
* Create one check for each type, using mostly the same code?

Reply:
@whitej6 WDYT?

Reply:
I am 50/50 on this. I see the benefit to have essentially N subclasses with an abstracted method on checking ancestor BUT that adds repetitive code.

Reply:
We just agreed to go with single checker for all types.